.golangci.yml

```diff
@@ -1,24 +1,20 @@
+version: "2"
 run:
-  timeout: 10m
   allow-parallel-runners: true
-
 linters:
-  disable-all: true
+  default: none
   enable:
     - asciicheck
     - bodyclose
+    - copyloopvar
     - dogsled
     - errcheck
-    - copyloopvar
     - goconst
     - gocritic
     - gocyclo
     - godot
-    - gofmt
```

Comment:
we need gofmt and goimports

Reply:
this is just the result of the auto migration https://golangci-lint.run/product/migration-guide/